external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSourceOffset.scala

```diff
@@ -19,7 +19,8 @@ package org.apache.spark.sql.kafka010
 
 import org.apache.kafka.common.TopicPartition
 
-import org.apache.spark.sql.execution.streaming.{Offset, SerializedOffset}
+import org.apache.spark.sql.execution.streaming.{Offset => OffsetV1, SerializedOffset}
```

Comment:
Ummm.. i think its better to rename new Offset to OffsetV2 than this rename old one to OffsetV1. This will keep it more consistent with other APIs which have V2 in them. Also, the MicroBatchExecution in the other PR also uses OffsetV2. Sorry for the nothing think ahead on this.